### PR TITLE
bump up deployment version in user-guide folder

### PR DESCRIPTION
**Pull request by nashasha1**
I will create a bunch of PRs for bumping up deployment version.  A pr contains all changes in a folder.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/7318)

### CODE REVIEW

**Comment by steveperry-53** [comment]
@nashasha1 I think a lot of these YAML files under user-guide are not used in our docs any more. So I think what we really should do is delete the files that are no longer in use. For example, if I go to docs/ and run `grep -r 'docs/user-guide/curlpod.yam.'`, I don't get any results.

**Comment by nashasha1** [comment]
@steveperry-53  That makes sense.  As you said, all the files which get none if I run  `grep -r '<path>/<filename>' ` can be deleted?

File: docs/user-guide/redis-deployment.yaml [hunk 1]
```diff
@@ -1,8 +1,11 @@
-apiVersion: apps/v1beta1
+apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: redis
 spec:
+  selector:
+    matchLabels:
+      app: redis
```
Comment (janetkuo) [inline docs/user-guide/redis-deployment.yaml]: and `tier: backend` (same comment for the other redis deployments)

Comment (nashasha1) [reply docs/user-guide/redis-deployment.yaml]: Done. Thank you for your reply.

**Comment by steveperry-53** [comment]
@nashasha1 The following files, under user-guide, are no longer used in the docs. Please use this PR to delete them. Thanks.

bad-nginx-deployment.yaml
curlpod.yaml
deployment.yaml
ingress.yaml
job.yaml
multi-pod.yaml
new-nginx-deployment.yaml
nginx-init-containers.yaml
nginx-lifecycle-deployment.yaml
nginx-probe-deployment.yaml
nginx-secure-app.yaml
nginx-svc.yaml
pod-w-message.yaml
pod.yaml
redis-deployment.yaml
redis-resource-deployment.yaml
redis-secret-deployment.yaml

**Comment by nashasha1** [comment]
@steveperry-53  Done!   Thank you for your reply. That's really helpful.

**Pull request merged**